themes/repair-cafe/partials/components/main-navigation.htm

```diff
@@ -6,7 +6,11 @@
 <div class="main-navigation">
     <nav class="navbar navbar-light navbar-expand-lg">
         <a class="navbar-brand d-lg-none" href="{{ 'homepage'|page }}">
-            <img src="{{ 'assets/images/repair-cafe-logo.jpg'|theme }}" height="30" />
+            {% if activeLocale == 'fr' or activeLocale == 'it' %}
```

Comment:
this will become ugly very soon, when more locales are added.

have you considered naming the default-logo without the locale, so as soon as the locale would be empty(if it would be empty anyways), it just uses that? not sure if that is more readable, but it would make adding a locale a bit easier, I think